src/Microsoft.ServiceFabric.Actors/Runtime/ActorGarbageCollectionSettings.cs

```diff
@@ -43,7 +43,7 @@ internal ActorGarbageCollectionSettings(ActorGarbageCollectionSettings settings)
         /// </exception>
         public ActorGarbageCollectionSettings(long idleTimeoutInSeconds, long scanIntervalInSeconds)
         {
-            // Verify that values are within acceptable range.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.